service/dps/storm/topologies/validation/src/test/java/eu/europeana/cloud/service/dps/storm/topologies/validation/topology/ValidationTopologyTest.java

```diff
@@ -214,6 +215,7 @@ private static void buildTopology() {
         ReadDatasetBolt readDataSetBolt = new ReadDatasetBolt(MCS_URL);
         ReadRepresentationBolt readRepresentationBolt = new ReadRepresentationBolt(MCS_URL);
         NotificationBolt notificationBolt = new NotificationBolt("", 1, "", "", "");
+        StatisticsBolt statisticsBolt = new StatisticsBolt("", 1, "", "", "");
         TestInspectionBolt endTest = new TestInspectionBolt();
```

Comment:
This test doesn't really tests the statistics as part of the validation topology, as it only assert expected notification tuples, There is nothing about the data inserted into the statistics tables.

Reply:
Tests with data are in StatisticsBoltTest. It should be enough as we agreed.